core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -510,9 +511,16 @@
    * LOAD_STATUS SUCCESS
    */
   public static final String STORE_LOADSTATUS_SUCCESS = "Success";
+
+  /**
+   * Default batch for data read in Columnar format
```

Comment:
we set the batch size = 4096 because vectorizedcarbonrecordreader in carbondata is using ColumnarBatch of defaultsize=4096 to read column batches,so we take it as standard